tools/accuracy_checker/accuracy_checker/annotation_converters/action_recognition.py

```diff
@@ -67,11 +70,14 @@ def configure(self):
         self.subset = self.get_value_from_config('subset')
         self.dataset_meta = self.get_value_from_config('dataset_meta_file')
         self.numpy_input = self.get_value_from_config('numpy_input')
+        self.image_input = self.get_value_from_config('image_input')
```

Comment:
do you have check that at least one input provided

Reply:
there is no check in current realization. if provided both inputs or no one, two-stream will be processed
there is no check in current implementation. if both inputs are provided or no one the rgb and flow will be processed